ludwig/features/text_feature.py

```diff
@@ -183,42 +184,30 @@ def add_feature_data(
 
 
 class TextInputFeature(TextBaseFeature, SequenceInputFeature):
-    def __init__(self, feature):
-        super().__init__(feature)
-
-        self.type = TEXT
+    encoder = 'parallel_cnn'
 
+    def __init__(self, feature, encoder_obj=None):
+        TextBaseFeature.__init__(self, feature)
+        SequenceInputFeature.__init__(self, feature)
         self.level = 'word'
         self.length = 0
+        self.overwrite_defaults(feature)
+        if encoder_obj:
+            self.encoder_obj = encoder_obj
+        else:
+            self.encoder_obj = self.initialize_encoder(feature)
 
-        self.encoder = 'parallel_cnn'
-
-        encoder_parameters = self.overwrite_defaults(feature)
-        self.encoder_obj = self.get_sequence_encoder(encoder_parameters)
+    def call(self, inputs, training=None, mask=None):
+        assert isinstance(inputs, tf.Tensor)
+        assert inputs.dtype == tf.int8
```

Comment:
this one should be any \int\, 8 16 32 64.

Reply:
That was my doubt, but couldn't generate anything else than 8. Thanks